### DISCUSSION TITLE
How can I blur my images?

### DISCUSSION THREAD

**Discussion by adampatterson** [imgproxy/imgproxy#681] [Q&A] [answered]
So far I love Imgproxy! 

I set it up yesterday and am working on building a utility library to manipulate my images.

I noticed in a few [examples](https://github.com/imgproxy/imgproxy/blob/master/docs/generating_the_url_basic.md#example) that the URL will use [colons](https://github.com/imgproxy/imgproxy/blob/master/docs/generating_the_url_advanced.md#example) with the params.

My URLs look something like this `/fill/900/600/no/0/` where I see `resize:fill`  or `gravity:ea`

adding `:bl:2.0` or just `2.0` produces an invalid URL.

I must be missing a configuration option but I'm not sure what it is.

Thanks for the help!

**Comment by DarthSim** [comment, marked as answer]
Hey Adam!

The basic URL format you use is very limited and doesn't support all the processing options imgproxy provides. More of all, I'm going to remove it from v3. In order to use the blur filter, you should use the [advanced URL format](https://docs.imgproxy.net/generating_the_url_advanced).

**Reply by adampatterson** [reply]
Thank you for the clarification.

I will update my code to use the key:value then.
